app/src/main/java/org/p2p/wallet/swap/ui/jupiter/tokens/presenter/SwapTokensPresenter.kt

```diff
@@ -0,0 +1,24 @@
+package org.p2p.wallet.swap.ui.jupiter.tokens.presenter
+
+import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.swap.ui.jupiter.tokens.SwapTokensContract
+import org.p2p.wallet.utils.Base58String
+
+class SwapTokensPresenter(
+    private val tokenToChange: SwapTokensChangeToken,
+    private val mapper: SwapTokensMapper
+) : BasePresenter<SwapTokensContract.View>(), SwapTokensContract.Presenter {
+
+    override fun attach(view: SwapTokensContract.View) {
+        super.attach(view)
+        view.setTokenItems(
+            mapper.toCellItems(Base58String(""), emptyList(), emptyList())
+        )
+    }
+
+    override fun onSearchTokenQueryChanged(newQuery: String) {
```

Comment:
= Unit

Reply:
no need for that now, this methods will be changed soon